client/blocks/nps-survey/index.jsx

```diff
@@ -0,0 +1,112 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import RecommendationSelect from './recommendation-select';
+
+class NpsSurvey extends Component {
+	static propTypes = {
+		onDismissed: PropTypes.func,
+		name: PropTypes.string
+	}
+
+	constructor( props ) {
+		super( props );
+		this.state = {
+			recommendationValue: null,
+			isSubmitting: false,
+			isSubmitted: false
+		};
+		this.handleRecommendationSelectChange = this.handleRecommendationSelectChange.bind( this );
+		this.handleFinishClick = this.handleFinishClick.bind( this );
+		this.handleDismissClick = this.handleDismissClick.bind( this );
+	}
+
+	handleRecommendationSelectChange( newRecommendationValue ) {
+		this.setState( {
+			recommendationValue: newRecommendationValue
```

Comment:
I'd like to keep the parseInt as low in the code as possible (in `RecommendationOption`), allowing all the rest of the code to just assume it is dealing with ints.